docs/installing.rst

```diff
@@ -27,21 +28,54 @@ In your ``conf.py`` file:
     :ref:`supported-dependencies`
         Officially Supported versions of Python, Sphinx, and other dependencies.
 
+
 .. note::
 
    Adding this theme as an extension is what enables localization of theme
    strings in your translated output. If these strings are not translated in
    your output, either we lack the localized strings for your locale, or you
    are using an old version of the theme.
 
+   ..
+      comment about this note: it's possibly not necessary to add the theme as an extension.
+      Rather, this is an issue caused by setting html_theme_path.
+      See: https://github.com/readthedocs/readthedocs.org/pull/9654
+
+
+.. _howto_upgrade:
+
+How to upgrade
+--------------
+
+.. tip:: 
+    **Read the Docs users:** To know which version of ``sphinx-rtd-theme`` you are running, you can always check your latest build outputs. One or more steps in the documentation build will install python packages, one of which is ``sphinx-rtd-theme``.
+
+Adding ``sphinx-rtd-theme`` to your project's ``requirements.txt`` will make pip install the latest compatible version of the theme.
```

Comment:
I like to be inclusive for project types when discussing dependency installation -- a good chunk of projects use packaging mechanisms besides requirements files.

```suggestion
Adding ``sphinx-rtd-theme`` to your project's dependencies will make pip install the latest compatible version of the theme.
```